core/src/main/java/feast/core/model/FeatureSet.java

```diff
@@ -121,6 +121,17 @@ private String getProjectName() {
     }
   }
 
+  public boolean hasAllLabels(Map<String, String> labels) throws InvalidProtocolBufferException {
+    Map<String, String> featureSetLabelsMap = this.toProto().getSpec().getLabelsMap();
```

Comment:
I think you should implement a `getLabelsMap` on this class instead of converting to proto here.

Reply:
Updated.